docs/donor/README.md

```diff
@@ -0,0 +1,115 @@
+# donor
+
+Related files:
+
+- [📝 Excel template](https://raw.githubusercontent.com/hubmapconsortium/ingest-validation-tools/master/docs/donor/donor.xlsx): For metadata entry.
+- [📝 TSV template](https://raw.githubusercontent.com/hubmapconsortium/ingest-validation-tools/master/docs/donor/donor.tsv): Alternative for metadata entry.
+
+This file is really only here to provide descriptions of fields in portal-ui. The full description of Donor metadata is at https://portal.hubmapconsortium.org/docs/donor
+Most definitions taken from https://ncit.nci.nih.gov/ncitbrowser/pages/home.jsf?version=20.11e
+
+
+
+## Metadata schema
+
+
+<details open="true"><summary><b>Version 0 (current)</b></summary>
+
+<blockquote>
+
+[`age_value`](#age_value)<br>
+[`blood_type`](#blood_type)<br>
+[`body_mass_index_value`](#body_mass_index_value)<br>
+[`cause_of_death`](#cause_of_death)<br>
+[`height_unit`](#height_unit)<br>
+[`kidney_donor_profile_index_value`](#kidney_donor_profile_index_value)<br>
+[`mechanism_of_injury`](#mechanism_of_injury)<br>
+[`medical_history`](#medical_history)<br>
+[`race`](#race)<br>
+[`sex`](#sex)<br>
+[`weight_value`](#weight_value)<br>
+
+</blockquote>
+
+##### `age_value`
+The time elapsed since birth.
+| constraint | value |
+| --- | --- |
+| type | `number` |
+| required | `True` |
+
+##### `blood_type`
+ABO blood type or "serotype" refers to the presence/absence of the either/both A & B blood antigens.
+| constraint | value |
+| --- | --- |
+| enum | `A`, `B`, `AB`, or `O` |
+| required | `True` |
+
+Ontology terms:
+
+| term | URI |
+| --- | --- |
+| A | `http://purl.bioontology.org/ontology/SNOMEDCT/49790006` |
```

Comment:
I am not sure if we should include the definitions of the terms here or not. It would save submitters one click but would be a lot of work to maintain this if we include it here. 

What was your rationale for not including a definition?

Reply:
Summarizing the discussion yesterday:

- We'd like to utilize IEC's knowledge graph here: Either they provide an API we can hit at doc generation time to get a definition for a term, or, simpler, we just generate a link parameterized by the URI to a human-readable page supported by the HuBMAP ontology service.
- We **should not** link to the uri, or attempt to use HTTP on it.
  - wikipedia: ["Other URIs provide only a unique name, without a means of locating or retrieving the resource or information about it"](https://en.wikipedia.org/wiki/Uniform_Resource_Identifier)
  - w3c: ["The argument that `http:` URIs are "locations" is based upon incomplete understanding of the use of URIs."](https://www.w3.org/2001/tag/doc/URNsAndRegistries-50.html#iddiv2137969152)
  - SO: ["in general all URI cannot be referenced"](https://stackoverflow.com/questions/1115503/use-of-uri-and-url-in-semantic-web)

Reply:
Are there any recommendations on how to resolve ontology URIs?

Reply:
Eventually, the knowledge graph API, but that doesn't exist right now.